From 0fba13c7fb502568c38de99ba41719c7b4e4fd9d Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 15 Jan 2019 10:23:13 +0200
Subject: [PATCH] [ozone/wayland]: fix vaapi build.

This is a fix to fix vaapi build with ozone wayland. It's not
tested, but requested by external.

Issue: https://github.com/Igalia/chromium/issues/511
Change-Id: Id3cf0702e9ea6c93fef0f99c23029ab2b1b291d2
---
 media/gpu/vaapi/BUILD.gn | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/gpu/vaapi/BUILD.gn b/media/gpu/vaapi/BUILD.gn
index cf471b7feb856..1c66745954048 100644
--- a/media/gpu/vaapi/BUILD.gn
+++ b/media/gpu/vaapi/BUILD.gn
@@ -88,12 +88,12 @@ source_set("vaapi") {
     ]
   }
 
-  if (ozone_platform_gbm || use_egl) {
+  if (ozone_platform_wayland || ozone_platform_gbm || use_egl) {
     sources += [
       "vaapi_picture_native_pixmap.cc",
       "vaapi_picture_native_pixmap.h",
     ]
-    if (ozone_platform_gbm) {
+    if (ozone_platform_gbm || ozone_platform_wayland) {
       sources += [
         "vaapi_picture_native_pixmap_ozone.cc",
         "vaapi_picture_native_pixmap_ozone.h",
